app/code/Magento/Store/Test/Mftf/Test/AdminDeleteStoreGroupTest.xml

```diff
@@ -44,11 +44,11 @@
         </actionGroup>
 
         <!--Go to backup index page and verify AssertBackupInGrid-->
-        <amOnPage url="{{BackupIndexPage.url}}" stepKey="goToBackupIndexPage"/>
+        <amOnPage url="{{AdminBackupIndexPage.url}}" stepKey="goToBackupIndexPage"/>
         <waitForPageLoad stepKey="waitForBackupIndexPageLoad"/>
```

Comment:
Replace with `AdminBackupIndexPageOpenActionGroup` or something similar (I haven't checked if this exists)

Reply:
I also not able to understand why we need to replace the page URL with Action group. Can you please explain?
Thanks

Reply:
Because tests should be made of reusable code blocks, thus Action Groups.
Then when you need to change behaviour by - for example - adding extra wait / click / whatever, before or after entering the page, it's enough to customize single action group and all the tests that follow this practice, get the change immediately.

Reply:
Thanks for the explanation